app/src/pages/summary.tsx

```diff
@@ -37,7 +40,41 @@ const Summary: NextPage = () => {
   )
 }
 
-export const getServerSideProps: GetServerSideProps = async ({ locale }) => {
+export const getServerSideProps: GetServerSideProps = async ({
+  locale,
+  req,
+  res,
+}) => {
+  const sessionCookie: string = getCookie('session', { req, res }) as string
+  const session: DefaultState = JSON.parse(sessionCookie) as DefaultState
+  const householdSize = session?.householdSize
+
+  const request = await fetch(
+    `${process?.env?.HOST || ''}/v1/eligibility-screener`,
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        first_name: session?.contact.firstName,
+        last_name: session?.contact.lastName,
+        phone_number: session?.contact.phone,
+        eligibility_categoies: session?.csv?.category_copy_array,
+        has_prior_wic_enrollment:
+          session?.eligibility?.before.replace(/[2]/g, '') === 'yes',
+        eligibility_programs: session?.csv?.program_copy_array,
+        household_size: householdSize && parseInt(householdSize),
+        zip_code: '11111',
+        wic_clinic: session?.clinic?.clinic,
+        applicant_notes: session?.contact.other,
+      }),
+      headers: {
+        'Content-type': 'application/json',
+        'X-Auth': process.env.API_AUTH_TOKEN || '',
+      },
+    }
+  )
+  const apiResponse: object = (await request.json()) as object
```

Comment:
I think this is on the right track, but it doesn't handle any non-happy paths, such as:
- what happens if the environment variables are empty?
- what happens if session is empty or it's missing some fields that the API requires for this post to succeed?
- what happens if the API returns something other than 200?

We also definitely need tests for key functionality like this.

Reply:
@rocketnova I agree -- was just trying to get an MVP out since I knew that I was going to refactor this next sprint anyway. I figured it would be easier to test post-refactor as well. Testing an API call made from `getServersideProps` didn't seem the most intuitive to me, I asked Tomas for some guidance on testing this kind of work though and was planning on collaborating with him on this during the refactor.